aptos-move/framework/aptos-framework/sources/account/account_abstraction.move

```diff
@@ -43,9 +45,9 @@ module aptos_framework::account_abstraction {
         V1 { auth_functions: OrderedMap<FunctionInfo, bool> }
     }
 
-    /// Update dispatchable authenticator that enables account abstraction.
+    /// Add dispatchable authentication function that enables account abstraction via this function.
     /// Note: it is a private entry function that can only be called directly from transaction.
-    public entry fun add_dispatchable_authentication_function(
+    entry fun add_authentication_function(
         account: &signer,
         module_address: address,
         module_name: String,
```

Comment:
This is Add or Update correct?  Would be good to put it in the comment

Reply:
It is add only.